### ISSUE TITLE
Plots backend documentation not reproducible/unclear

### ISSUE THREAD

**Issue by japhir** [JuliaLang/julia#53229]
Hi! New Julia learner here!

I've just tried to reproduce [this section of the manual](https://docs.juliaplots.org/latest/backends/#Persistent-backend-selection) but I get the following error:
```
$ JULIA_PKG_PRECOMPILE_AUTO=0 julia -e 'import Plots; Plots.set_default_backend!(:pythonplot)'
ERROR: UndefVarError: `set_default_backend!` not defined
Stacktrace:
 [1] top-level scope
   @ none:1
```

I found the Plots backend section a little hard to follow: I wanted to understand how to create an interactive version of a plot, which lead me to conclude that I should switch the backend.

> Also, PlotlyJS supports saving the output to more formats than Plotly, such as EPS and PDF, and thus is the recommended version of Plotly for developing publication-quality figures.

made me think I should try this out, but I didn't see any docs on how to change the backend temporarily in the docs. Only the above permanent change (that didn't work).

I hope this is a helpful issue!

EDIT: I've also installed Preferences, since the top of the page mentions that this mechanism is used. Still no luck though.

```
versioninfo()
Julia Version 1.10.0
Commit 3120989f39 (2023-12-25 18:01 UTC)
Build Info:

    Note: This is an unofficial build, please report bugs to the project
    responsible for this build and not to the Julia project unless you can
    reproduce the issue using official builds available at https://julialang.org/downloads

Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: 4 × Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-15.0.7 (ORCJIT, skylake)
  Threads: 1 on 4 virtual cores
```

I've installed julia through the pacman package manager for Arch Linux.

**Comment by jishnub** [comment]
Hi, this should be reported to Plots.jl, so that their documentation may be improved. That project is developed independent of julia.
